plugins/package-managers/python/src/funTest/assets/projects/external/spdx-tools-python-expected-output.yml

```diff
@@ -28,44 +28,39 @@ project:
     - id: "PyPI::ply:3.11"
     - id: "PyPI::rdflib:5.0.0"
       dependencies:
-      - id: "PyPI::isodate:0.6.1"
-        dependencies:
-        - id: "PyPI::six:1.16.0"
+      - id: "PyPI::isodate:0.7.0"
       - id: "PyPI::pyparsing:2.4.7"
       - id: "PyPI::six:1.16.0"
 packages:
-- id: "PyPI::isodate:0.6.1"
-  purl: "pkg:pypi/isodate@0.6.1"
+- id: "PyPI::isodate:0.7.0"
+  purl: "pkg:pypi/isodate@0.7.0"
   authors:
-  - "Gerhard Weis <gerhard.weis@proclos.com>"
+  - "Gerhard Weis"
   declared_licenses:
-  - "BSD"
   - "BSD License"
   declared_licenses_processed:
-    spdx_expression: "BSD-3-Clause"
-    mapped:
-      BSD: "BSD-3-Clause"
-      BSD License: "BSD-3-Clause"
+    unmapped:
+    - "BSD License"
   description: "An ISO 8601 date/time/duration parser and formatter"
-  homepage_url: "https://github.com/gweis/isodate/"
+  homepage_url: ""
   binary_artifact:
-    url: "https://files.pythonhosted.org/packages/b6/85/7882d311924cbcfc70b1890780763e36ff0b140c7e51c110fc59a532f087/isodate-0.6.1-py2.py3-none-any.whl"
+    url: ""
     hash:
-      value: "0751eece944162659049d35f4f549ed815792b38793f07cf73381c1c87cbed96"
-      algorithm: "SHA-256"
+      value: ""
+      algorithm: ""
```

Comment:
Any idea why we don't get binary artifact information anymore?

Reply:
I quickly debugged this and python-inspector really only seems to report the source package here now.

Reply:
Although https://pypi.org/project/isodate/#files has the WHL listed...